src/main/java/winstone/cmdline/Option.java

```diff
@@ -58,6 +58,7 @@
     public static final OInt HTTP_PORT=integer("http"+_PORT,8080);
     public static final OString HTTP_LISTEN_ADDRESS=string("http"+ _LISTEN_ADDRESS);
     public static final OInt HTTP_KEEP_ALIVE_TIMEOUT=integer("http" + _KEEP_ALIVE_TIMEOUT, _KEEP_ALIVE_TIMEOUT.defaultValue);
+    public static final OBoolean HTTP_HTTPS_REDIRECT=bool("https_redirect", false);
```

Comment:
is it used?

Reply:
With all my changes I forgot to add the use for it :-( now it is added

Reply:
Thank you :-)